integr8ly/cloud-resource-operator
# Go vesion bumped to 1.16.2

## Overview
Go version used was updated to 1.16.2

## Verification
- Confirm that e2e tests were successful
- Install operator on the byoc cluster using make commands and run the set of tests by using the `make cluster/seed commands`
